app/views/ships/show.html.erb

```diff
@@ -9,7 +9,6 @@
 
 <table id="coord_button"> 
 	<tr>
-		<td class="userbutton"> <%= link_to 'Edit', edit_ship_path(@ship) %></td>
```

Comment:
Der Button ist ausgelagert in Name ändern. Damit besser zu erkennen ist, was wo gemacht werden kann